docs/translations.md

```diff
@@ -6,7 +6,7 @@ Before changing or adding languages, read the sections below to understand the m
 
 ## Translation status
 
-Here is a list of all supported translations included in the OTT Web App. The status indicates if the translations are 
+Here is a list of all supported translations included in the OTT Web App. The status indicates if the translations are
```

Comment:
@ChristiaanScheermeijer do you think we should move this file to the `web` folder? Or do you have plans to move translations out of the web to reuse in other implementations?

Reply:
Here is the answer probably:

```
> The translation files will be moved to the `./packages/i18n` package in the near future.
```